src/components/Search.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <input name="search" v-model="value" /><button
+      name="search"
+      @click="search"
+    >
+      Search
+    </button>
+    <ol>
+      <li v-for="match of matches" @click="choose(match)">
+        {{ match.title }}
+        {{ match.id }}
+        <img
+          :src="
+            'https://media.getty.edu/iiif/image/' +
+              match.manifest +
```

Comment:
manifest is probably a confusing name here--it's the UUID that's part of the image ID, and not a manifest at all?